drivers/sensor/bh1750/Kconfig

```diff
@@ -0,0 +1,50 @@
+# BH1750 ambient light sensor configuration options
+
+# Copyright (c) 2023 Sngular People SL.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+DT_COMPAT_ROHM_BH1750 := rohm,bh1750
```

Comment:
Don't need this anymore. Please remove

Reply:
Thank you for your kind words!

Ok, I delete it